ui/packages/catalog-ui-search/src/main/webapp/component/paging/paging.hbs

```diff
@@ -20,7 +20,7 @@
 <div class="buttons-left">
 {{#if showPreviousServerPage}}
     <button class="server-previous is-primary">
-        <span class="fa fa-angle-double-left"/>
+        <span class="fa fa-angle-double-left"></span>
```

Comment:
The self-closing tag was causing the "Back" span to be nested within this span for some reason, causing the `fa` class to apply a different `font-family` to the "Back" text. Explicitly closing the span prevents this.